app/models/user.rb

```diff
@@ -151,6 +151,7 @@ class User < ApplicationRecord
   before_validation :set_config_input
   before_validation :downcase_email
   before_validation :check_for_username_change
+  before_validation :evaluate_markdown, if: :summary
```

Comment:
Hi, the problem with this, if I'm not mistaken is that you'll end up with a populated HTML without a summary, in the following scenario:

empty summary -> filled summary generates a HTML -> summary emptied, doesn't empty the HTML.